### ISSUE TITLE
About the triplet loss

### ISSUE THREAD

**Issue by wangjiyuan9** [RuijieZhu94/EC-Depth#2]
Hello, Jerry！
At [here](https://github.com/RuijieZhu94/EC-Depth/blob/d37575b5cc251d2285ebe94ccb1f919c212338c0/trainer.py#L708C13-L724C79), the paper had mentioned the 'Perturbation-invariant depth consistency loss'. But your train_first_stage.sh didn't have the "--use_triplet_loss".
what's more,for :
 disp_mix = torch.clamp((disp + disp_aug1 + disp_aug2) / 3., 1e-7, 1).**log()**
why there is a log()?

**Comment by RuijieZhu94** [comment]
Hi, Jiyuan. Thank you for correction. The train_first_stage.sh has already updated.
For your question, the KL divergence requires a log value as input.
You can find the detailed use from this [url](https://pytorch.org/docs/stable/generated/torch.nn.KLDivLoss.html#torch.nn.KLDivLoss).
Hope this answer is helpful to you.
